RubberduckTests/Inspections/AssignedByValParameterInspectionTests.cs

```diff
@@ -97,19 +97,152 @@ public void AssignedByValParameter_ReturnsResult_SomeAssignedByValParams()
         [TestCategory("Inspections")]
         public void AssignedByValParameter_QuickFixWorks()
```

Comment:
Huh, you're right - nevermind that. We only care about compilable code... sorry for that misleading comment, you can drop that test.